octodns/manager.py

```diff
@@ -421,16 +466,25 @@ def validate_configs(self):
         for zone_name, config in self.config['zones'].items():
             zone = Zone(zone_name, self.configured_sub_zones(zone_name))
 
+            source_zone = config.get('alias')
+            if source_zone:
+                if source_zone not in self.config['zones']:
+                    self.log.exception('Invalid alias zone')
+                    raise ManagerException('Invalid alias zone {}: '
+                                           'source zone {} does not exist'
+                                           .format(zone_name, source_zone))
+                continue
```

Comment:
Yeah, this definitely appears to be https://github.com/nedbat/coveragepy/issues/198#issuecomment-399705984. Surprised I haven't run into this before. There's a pragma for coverage, but we've disabled the use of it b/c things slipped through in the past that were actually bugs/problems where it was used. 

The best short-term work-around I can think of is something like this:

```suggestion
                # this is just here to satisfy coverage, see
                # https://github.com/nedbat/coveragepy/issues/198
                source_zone = source_zone
```

I may look into a clean way to do the pragma and allow it "just this once" in the future, but for now that should do the trick without any real side effects :-/

Reply:
Woops sorry, I missed your comment. I will fix that.

Reply:
Hmmm... All checks have passed so I don't know if I should apply your change or not?

Reply:
Please apply it. It's a shortcoming of the check for the pragma. It looks for `pragma: nocover` and doesn't consider the case where it's `pragma: no cover` with a space. That's by accident and it'll get fixed the next time I'm in there.

Reply:
Done.